Clarify sentence regarding default implementation (#212)
vtex-apps/minicart: docs/README.md

By default implementation we mean that whenever you use `minicart.v2` in your store you're actually using the `json` above behind the scenes.

> Therefore, in order to customize the Minicart configuration, you can simply use the default implementation in your code instead of just declaring the `minicart` and change it as you wish.

Reviewer: ```suggestion
Therefore, in order to customize the Minicart configuration, you can simply copy the code above and change it as you wish.
```
